Maple2.Server.Game/Model/Field/Actor/FieldPlayer.cs

```diff
@@ -20,20 +22,31 @@ public class FieldPlayer : Actor<Player> {
     private long battleTick;
     private bool inBattle;
 
+    #region DebugFlags
     private bool debugAi = false;
+    public bool DebugSkills = false;
+    #endregion
 
     public int TagId = 1;
 
     private readonly EventQueue scheduler;
 
-    public FieldPlayer(GameSession session, Player player) : base(session.Field!, player.ObjectId, player) {
+    public FieldPlayer(GameSession session, Player player, NpcMetadataStorage npcMetadata) : base(session.Field!, player.ObjectId, player, GetPlayerModel(player.Character.Gender), npcMetadata) {
         Session = session;
 
         scheduler = new EventQueue();
         scheduler.ScheduleRepeated(() => Field.Broadcast(ProxyObjectPacket.UpdatePlayer(this, 66)), 2000);
         scheduler.Start();
     }
 
+    private static string GetPlayerModel(Gender gender) {
+        return gender switch {
+            Gender.Male => "male",
+            Gender.Female => "female",
+            _ => ""
```

Comment:
maybe we should just default this to male? Just to be safe? I don't forsee this ever being an issue but it couldn't hurt